src/ir_Daikin.cpp

```diff
@@ -2322,34 +2322,38 @@ void IRDaikin176::off(void) { setPower(false); }
 /// @param[in] on true, the setting is on. false, the setting is off.
 void IRDaikin176::setPower(const bool on) {
   remote_state[kDaikin176ByteModeButton] = 0;
-  setBit(&remote_state[kDaikin176BytePower], kDaikinBitPowerOffset, on);
+  setBit(&remote_state[kDaikin176ByteModePower], kDaikinBitPowerOffset, on);
 }
 
 /// Get the value of the current power setting.
 /// @return true, the setting is on. false, the setting is off.
 bool IRDaikin176::getPower(void) {
-  return GETBIT8(remote_state[kDaikin176BytePower], kDaikinBitPowerOffset);
+  return GETBIT8(remote_state[kDaikin176ByteModePower], kDaikinBitPowerOffset);
 }
 
 /// Get the operating mode setting of the A/C.
 /// @return The current operating mode setting.
 uint8_t IRDaikin176::getMode(void) {
-  return GETBITS8(remote_state[kDaikin176ByteMode], kHighNibble, kModeBitsSize);
+  return GETBITS8(remote_state[kDaikin176ByteModePower], kHighNibble,
+                  kModeBitsSize);
 }
 
 /// Set the operating mode of the A/C.
 /// @param[in] mode The desired operating mode.
 void IRDaikin176::setMode(const uint8_t mode) {
   uint8_t altmode = 0;
   switch (mode) {
-    case kDaikinFan: altmode = 0; break;
-    case kDaikinDry: altmode = 7; break;
-    case kDaikin176Cool: altmode = 2; break;
-    default: this->setMode(kDaikin176Cool); return;
+    case kDaikin176Dry:  altmode = 2; break;
```

Comment:
Never noticed this altmode before, quite magic without comments?

Reply:
Fair call. Added some comments to document it.